modules/release-notes/partials/docs-server-7.6-release-note.adoc

```diff
@@ -73,9 +165,21 @@ This release contains the following fixes.
 |Issue | Description | Resolution
 
 | https://issues.couchbase.com/browse/MB-39484[MB-39484]
-| Remove N1QL support for password-less buckets.
+| Remove {sqlpp} support for password-less buckets.
 | Password-less bucket support removed.
 
+| https://issues.couchbase.com/browse/MB-52228[MB-52228]
+| CREATE INDEX fails when the index key expression matches the keyspace name.
+| If the index key expression matches the keyspace name, the index key expression is no longer converted to SELF.
+
+| https://issues.couchbase.com/browse/MB-60762[MB-60762]
```

Comment:
This one has been reopened & reverted AFAIK, so probably shouldn't be included yet.